src/app/add-log/complete.tsx

```diff
@@ -0,0 +1,18 @@
+export default function StepComplete({ onRestart }: { onRestart: () => void }) {
+  return (
+    <div className="flex h-dvh flex-col justify-between bg-[#F4F6F9] pt-[16px] px-[24px] pb-[42px]">
+      <div className="mb-[24px]">
+        <p className="text-subtitle-1-b ">이제 물 섭취량을</p>
+        <p className="text-subtitle-1-b">기록할 수 있어요.</p>
+      </div>
```

Comment:
이런 식이 더 깔끔해 보이네요~! p 태그를 썻지만 한개의 태그가 한개의 내용을 담고 있는게 아니라 개행의 용도라면 `<br />`을 쓰는건 어떨까요?

```suggestion
      <h1 className="mb-6 text-subtitle-1-b">
        이제 물 섭취량을<br />
        기록할 수 있어요.
      </h1>
```